File: Examples/DotNET/SDK/VisualBasic/Fields/UpdateFields.vb
Imports Aspose.Cloud

Namespace Fields
    Class UpdateFields
        Public Shared Sub Run()
            Dim inputfile As String = "doc-sample.doc"

            ' Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, String.Empty)

            ' Update All Fields in the Word Document
            Dim wordsDocumentResponse As WordsDocumentResponse = Common.WordsService.WordsDocumentFields.UpdateFieldsInDocument(inputfile, Common.GetOutputFilePath(inputfile, True), Common.FOLDER)

            Console.WriteLine(vbLf & "All fields updated successfully.")
        End Sub
    End Class
End Namespace

File: Examples/DotNET/SDK/CSharp/Fields/UpdateFields.cs
using System;
using Aspose.Cloud;

namespace CSharp.Fields
{
    class UpdateFields
    {
        public static void Run()
        {          
            string inputfile = "doc-sample.doc";

            // Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, string.Empty);

            // Update All Fields in the Word Document
            WordsDocumentResponse wordsDocumentResponse = Common.WordsService.WordsDocumentFields.UpdateFieldsInDocument(inputfile, Common.GetOutputFilePath(inputfile, true), Common.FOLDER);
            
            Console.WriteLine("\nAll fields updated successfully.");
        }
    }
}
